charts/zalenium/values.yaml

```diff
@@ -143,15 +143,20 @@ persistence:
     size: 80Gi
 
 ingress:
-  enabled: false
+  enabled: true
```

Comment:
Most of the public chart tend to disable it by default as in some cloud provider, it is just easier to play with a service of type `LoadBalancer`. I think it is better to follow this kind of guideline.